pages/changelog.mdx

```diff
@@ -4,6 +4,85 @@ All notable changes to Craftable PRO will be documented on this page.
 
 <div className="changelog-container">
 
+## v1.1.21 - 2023-10-24
+
+### What's Changed
+
+- fix: show validation message in invite user
+- feat: added casting to generator/model
+- fix: changed redirect() to Inertia-location
+- fix: deleted unused directive
+- fix: pagination (delete query page)
+- fix: default route
+- build(deps-dev): bump postcss from 8.4.20 to 8.4.31
```

Comment:
we don't need these here, its not really important since we don't use npm packages anywhere